### ISSUE TITLE
New command for calldata encoding

### ISSUE THREAD

**Issue by gitpusha** [OpenZeppelin/openzeppelin-sdk#1263]
TL;DR:

- CLI command that abi encodes args you pass
- CLI command that abi encodes function Signature you pass with its args

It would be sweet to have a `oz` `CLI` `command` `option` that let's you pass `args` and have them `abi` `encoded` into a `bytes` `payload` by passing a `option` flag like `--bytes` or `--encode`. 
It would also be cool to have a `oz` `CLI` `command` `option` that let's you pass a `function Signature` with its `args` and then pass a `flag` like `--encodeWithSig`, so that the `payload` is automatically converted into `abi.encoded bytes` with a `function selector`.   

I use a lot of contracts that usually would get abi encoded parameters from a frontend. It would be sweet to be able to play around with these contracts from the `oz` `CLI` tool too, without having to manually encode the `args` I wanna pass to a `send-tx` myself.

I have seen such a `CLI` feature before with `seth`. Check it out:
http://dapp.tools/seth/

![image](https://user-images.githubusercontent.com/36712489/67467486-195bc180-f649-11e9-8e2d-65668df9a378.png)

**Comment by spalladino** [comment]
Possible related to https://github.com/OpenZeppelin/openzeppelin-sdk/issues/1179
